src/features/defi/components/Approve/Approve.tsx

```diff
@@ -36,9 +36,8 @@ type ApproveProps = {
   fiatEstimatedGasFee: string
   isExactAllowance?: boolean
   isApproved?: boolean
-  learnMoreLink?: string
   loading: boolean
-  loadingText?: string
```

Comment:
Effectively useless - `loadingText` was always the same, and it being omitted was a bug, not a feature.